### ISSUE TITLE
sfizz as external dynamic library

### ISSUE THREAD

**Issue by yurivict** [sfztools/sfizz-ui#135]
It says somewhere that in 2023 the sfizz project was split into sfizz and sfizz-ui.
But in fact it was renamed into sfizz-ui, and the sfizz module was split out and included from sfizz-ui.

It would be better, more modular, if sfizz-ui would be able to use the external sfizz library.
This way users who only need this library would be able to only install sfizz, and others would be able to install sfizz-ui on top of it.

Packagers are confused, and only one of the port systems has sfizz-ui: https://repology.org/projects/?search=sfizz

**Comment by redtide** [comment]
> It says somewhere that in 2023 the sfizz project was split into sfizz and sfizz-ui.
> But in fact it was renamed into sfizz-ui, and the sfizz module was split out and included from sfizz-ui.

No, sfizz-ui is a new repository, the git history is all in the original sfizz repository, all plugins related code was moved in this place.

> It would be better, more modular, if sfizz-ui would be able to use the external sfizz library.
This way users who only need this library would be able to only install sfizz, and others would be able to install sfizz-ui on top of it.

Agreed, I had already a [conversation](https://github.com/sfztools/sfizz-ui/issues/127#issuecomment-1952875504) about this with the Arch packager, but unfortunately it's not a that trivial thing for me.
